rotary/rules/rotary.smk

```diff
@@ -514,13 +514,13 @@ rule polish_polypolish:
 # TODO - the relative path workarounds in the shell here are a bit odd because polca outputs files in the present working directory
 rule polish_polca:
     input:
-        qc_short_r1 = "{sample}/{sample}_R1.fastq.gz",
-        qc_short_r2 = "{sample}/{sample}_R2.fastq.gz",
-        polished = "{sample}/polish/polypolish/polypolish.fasta"
+        qc_short_r1 = "{sample}/raw/{sample}_R1.fastq.gz",
+        qc_short_r2 = "{sample}/raw/{sample}_R2.fastq.gz",
+        polished = "{sample}/polish/polypolish/{sample}_polypolish.fasta"
     output:
-        polca_output = "{sample}/polish/polca/polca.fasta",
-        polypolish_sam = temp("{sample}/polish/polca/polypolish.fasta.unSorted.sam"),
-        polypolish_bam = temp("{sample}/polish/polca/polypolish.fasta.alignSorted.bam")
+        polca_output = "{sample}/polish/polca/{sample}_polca.fasta",
```

Comment:
Line 540 (symlink) will need to be changed to match this name change. I think I didn't use `{output.polca_output}` there because of the odd `cd` steps. However, I think line 540 should be changed to properly use `{output.polca_output}` instead of hard-coding the final name.

Reply:
I will check is `ln` allows longer paths.

Reply:
I think its more straightforward to add wildcards to this since there is a `cd`. Added.